src/core/a-entity.js

```diff
@@ -44,7 +44,9 @@ var proto = Object.create(ANode.prototype, {
       this.states = [];
       this.components = {};
       this.paused = true;
-      this.object3D = new THREE.Mesh();
+      this.object3D = new THREE.Object3D();
+      this.object3D.el = this;
+      this.objects3D = {};
```

Comment:
`object3Ds` sounds more like the plural form `object3D`. `objects3D` sounds like objects, but 3D.